.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Install rust toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
Because of `rustfmt` which produces the following error on a stable channel:
```
Warning: can't set `imports_granularity = Crate`, unstable features are only available in nightly channel.
```